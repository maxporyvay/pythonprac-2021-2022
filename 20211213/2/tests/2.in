[88, -4, 83, 58, 40, 39, 1, 96, 2, 8, 76, -9, 54, -1, 29, 88, 31, -9, 1, 90, 46, 50, 98, 23, 59, -10, 99, 33, -8, 25, 94, -10, 27, 26, 84, 54, 38, 26, 92, 18, 46, 66, 60, 71, 69, 55, 0, 71, 73, 43, 41, -10, -8, 56, -5, 81, 77, 64, 71, 32, 2, 56, -7, 49, 52]