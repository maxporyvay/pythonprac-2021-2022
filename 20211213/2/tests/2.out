[-10, -10, -10, -9, -9, -8, -8, -7, -5, -4, -1, 0, 1, 1, 2, 2, 8, 18, 23, 25, 26, 26, 27, 29, 31, 32, 33, 38, 39, 40, 41, 43, 46, 46, 49, 50, 52, 54, 54, 55, 56, 56, 58, 59, 60, 64, 66, 69, 71, 71, 71, 73, 76, 77, 81, 83, 84, 88, 88, 90, 92, 94, 96, 98, 99]
